spec/factories/auth_source_factory.rb

```diff
@@ -36,4 +36,8 @@
     port 225  # a reserved port, should not be in use
     attr_login 'uid'
   end
+
+  factory :dummy_auth_source, class: DummyAuthSource do
+    name 'DerpLAP'
```

Comment:
You have to manually run a rake tast (`rake ldap:create_dummy`) to activate this auth source.